pkg/base1/cockpit-components-listing.jsx

```diff
@@ -128,7 +129,7 @@ define([
             var count_display = null;
 
             var header_entries = this.props.columns.map(function(itm) {
-                if (typeof itm === 'string' || itm instanceof String || React.isValidElement(itm))
+                if (typeof itm === 'string' || itm === undefined || itm instanceof String || React.isValidElement(itm))
```

Comment:
> Empty is empty.

But `undefined` isn't.

> Also, we accept react elements - and react has the same behavior with undefined objects, they are treated as empty.

At least react-lite doesn't. It accepts `null`, but not `undefined`[1].

> What kind of bugs will be hard to catch?

If some function that is supposed to return an element erroneously returns nothing, we'll get an oops if we don't accept `undefined`. I think it's better to be explicit when we want empty elements and return `null`.

[1] https://github.com/Lucifier129/react-lite/blob/993fd6a844ef7a7b5f28304c3e6be5ccd67d76d5/examples/simple/react-lite.js#L413